xDripG5/Messages/CalibrationDataRxMessage.swift

```diff
@@ -18,5 +21,8 @@ struct CalibrationDataRxMessage: TransmitterRxMessage {
         guard data.starts(with: .calibrationDataRx) else {
             return nil
         }
+
+        glucose = data[11..<13].to(UInt16.self)
```

Comment:
You can use `toInt()` on this line like the one below. The type is inferred due to being declared above.

Reply:
Changed. Thanks.